project_task_analytic_tag/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 PESOL (http://pesol.es)
+#                Luis Adan Jimenez Hernandez (luis.jimenez@pesol.es)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Project Task Analytic Tag",
+    "summary": "Analytic tag in project task",
+    "version": "10.0.1.0.0",
+    "author": "PESOL",
+    "website": "http://pesol.es",
+    "category": "Analytic Accont",
+    "license": "AGPL-3",
+    "depends": [
+        'project',
+        "hr_timesheet"
```

Comment:
This is related to ` analytic_tag_dimension` right?
Does it make sense to add it as a dependency?
If not, at least that module should be mentioned in the README, as a way to use the information added here.